common.go

```diff
@@ -67,8 +67,8 @@ type FieldSort struct {
 	D Dir
 }
 
-// String returns a representation of Sort compatible with the format of mgo.
-func (s Sort) String() string {
+// Mgo returns a representation of Sort compatible with the format of mgo.
+func (s Sort) MgoFormat() []string {
```

Comment:
In the comment should say `MgoFormat returns a ...` instead of `Mgo returns a...`.

Anyway, imho, "ToList" or "List" would be better names. It's not a "special" format mgo asks for, just asks for a variable number of sorts (a ""list"").